analyzers/src/SonarAnalyzer.Common/Extensions/ISymbolExtensions.cs

```diff
@@ -48,5 +48,22 @@ internal static class ISymbolExtensions
         // https://www.jetbrains.com/help/resharper/Reference__Code_Annotation_Attributes.html#NotNullAttribute
         private static bool IsNotNullAttribute(AttributeData attribute) =>
             attribute.HasAnyName("ValidatedNotNullAttribute", "NotNullAttribute");
+
+        // https://github.com/dotnet/roslyn/blob/2a594fa2157a734a988f7b5dbac99484781599bd/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ISymbolExtensions.cs#L93
+        public static ImmutableArray<ISymbol> ExplicitOrImplicitInterfaceImplementations(this ISymbol symbol)
```

Comment:
I have created this trello card in How we work about the placement of the helpers:
https://trello.com/c/nw3yLmEY
See also my comment about coverage there.

Reply:
Consider also adding the complementing methods that are available in Roslyn, like `ImplicitInterfaceImplementations` and `ExplicitInterfaceImplementations`. We should only do this after the trello card is resolved.